### PR TITLE
 adds confirmation prompt at joingame()

### PR DESCRIPTION
**Pull request by Tetsukiba**

### CODE REVIEW

**Review by NataKilar — APPROVED** [review]

File: mods/persistence/modules/mob/new_player.dm [hunk 1]
```diff
@@ -86,6 +86,12 @@
 	if(!config.enter_allowed)
 		to_chat(src, SPAN_NOTICE("There is an administrative lock on entering the game!"))
 		return
+
+	switch(alert("Did you create a character?",,"Yes","No"))
```
Comment (NataKilar) [inline mods/persistence/modules/mob/new_player.dm]: Extraneous , here.

This is also in the wrong place, we should only add this alert if the player doesn't have a player in game, so move it down to the line above create_character() in this same proc.

I'd also prefer it to be something like "Are you sure you want to join the game with the character you've created? This cannot be undone!"

**Review by NataKilar — DISMISSED** [review]

File: mods/persistence/modules/mob/new_player.dm [hunk 2]
```diff
@@ -118,6 +119,11 @@
 		qdel(src)
 		return
 
+
+	switch(alert("Are you sure you want to join the game with the character you've created? This cannot be undone!",,"Yes","No"))
```
Comment (NataKilar) [inline mods/persistence/modules/mob/new_player.dm]: Still got an extra "," here before "Yes"

Comment (Tetsukiba) [reply mods/persistence/modules/mob/new_player.dm]: alert(Usr=usr,Message,Title,Button1="Ok",Button2,Button3)  if I removed the coma, wouldn't it use "yes" as the title argument?

Comment (NataKilar) [reply mods/persistence/modules/mob/new_player.dm]: Yes, so add a title for cleanliness I suppose. "Character Confirmation" or such would work.

File: mods/persistence/modules/mob/new_player.dm (hunk 1 above)
Comment (MistakeNot4892) [reply mods/persistence/modules/mob/new_player.dm]: I believe the extra `,` means the `alert()` won't have a title.

Comment (MistakeNot4892) [reply mods/persistence/modules/mob/new_player.dm]: Local man reads ten lines down and realizes someone else said the same thing three weeks ago.

**Review by Andrew-Fall — APPROVED** [review]

**Pull request merged**